pyvista/core/filters/image_data.py

```diff
@@ -761,14 +761,17 @@ def _check_fft_scalars(self):
                 '`numpy.complex128`.'
             )
 
-    def _flip_uniform(self, axis) -> 'pyvista.UniformGrid':
+    def _flip_uniform(self, axis) -> 'pyvista.ImageData':
         """Flip the uniform grid along a specified axis and return a uniform grid.
 
-        This varies from :func:`DataSet.flip_x` because it returns a UniformGrid.
+        This varies from :func:`DataSet.flip_x` because it returns a ImageData.
 
         """
         alg = _vtk.vtkImageFlip()
         alg.SetInputData(self)
         alg.SetFilteredAxes(axis)
         alg.Update()
         return wrap(alg.GetOutput())
+
+
+UniformGridFilters = ImageDataFilters
```

Comment:
Leaving in namespace -- since this is an abstract class, not issue deprecation warning